pkg/operator/status.go

```diff
@@ -213,6 +213,11 @@ func (optr *Operator) getOrCreateClusterOperator() (*osconfigv1.ClusterOperator,
 			Resource: "clusterroles",
 			Name:     "machine-api-controllers",
 		},
+		{
+			Group:    "rbac.authorization.k8s.io",
+			Resource: "roles",
+			Name:     "machine-api-controllers",
```

Comment:
there's no role with this name in current PR

Reply:
whoops :)